ocis/README.md

```diff
@@ -23,4 +23,60 @@ To authenticate the connection to the etcd registry, you have to set `ETCD_USERN
 
 ## Memory limits
 
-oCIS will automatically set the go native `GOMEMLIMIT` to `0.9`. To disable the limit set `AUTOMEMEMLIMIT=off`. For more information take a look at the official [Guide to the Go Garbage Collector](https://go.dev/doc/gc-guide).
\ No newline at end of file
+oCIS will automatically set the go native `GOMEMLIMIT` to `0.9`. To disable the limit set `AUTOMEMEMLIMIT=off`. For more information take a look at the official [Guide to the Go Garbage Collector](https://go.dev/doc/gc-guide).
+
+## Cli commands
+
+The ocis package offers a variety of cli commands to monitor or repair ocis installations. All these commands have a common parameter: `--basePath` (or `-p`). This needs to point to a storage provider. Examples are:
+```bash
+.ocis/storage/users # bare metal installation
+/var/tmp/ocis/storage/users # docker installation
+...
+```
+This value can vary depending on your ocis installation.
+
+### Backup Cli
+
+The backup command allows inspecting the consistency of an ocis storage:
+```
+ocis backup consistency -p /base/path/storage/users
+```
+
+This will check the consistency of the storage and output a list of inconsistencies. Inconsistencies can be:
+* Orphaned Blobs: A blob in the blobstore that is not referenced by any file metadata
+* Missing Blobs: A blob referenced by file metadata that is not present in the blobstore
+* Missing Nodes: A node that is referenced by a symlink but doesn't exist
+* Missing Link: A node that is not referenced by any symlink but should be
+* Missing Files: A node that is missing essential files (such as the `.mpk` metadata file)
+* Missing/Malformed Metadata: A node that doesn't have any (or malformed) metadata
```

Comment:
```suggestion
This will check the consistency of the storage and output a list of inconsistencies. Inconsistencies can be:

* **Orphaned Blobs**\
A blob in the blobstore that is not referenced by any file metadata.
* **Missing Blobs**\
A blob referenced by file metadata that is not present in the blobstore.
* **Missing Nodes**\
A node that is referenced by a symlink but doesn't exist.
* **Missing Link**\
A node that is not referenced by any symlink but should be.
* **Missing Files**\
A node that is missing essential files (such as the `.mpk` metadata file).
* **Missing/Malformed Metadata**\
A node that doesn't have any (or malformed) metadata.
```
This improves readbility